index.html

```diff
@@ -0,0 +1,263 @@
+            <i class="fas fa-laptop-code"></i>
+            <h3>Full Stack</h3>
+            <p>
+              Learn full stack web developer with html, css, javascript and
+              ruby.
+            </p>
+          </div>
+          <div class="border-box">
+            <i class="fa fa-bar-chart" aria-hidden="true"></i>
+            <h3>Data Science</h3>
+            <p>
+              Learn data science and do expert for analysis and make decision.
+            </p>
+          </div>
+          <div class="border-box">
+            <i class="fas fa-database"></i>
+            <h3>DataBase</h3>
+            <p>
+              Build strong database working with server and all required from
+              web or data science.
+            </p>
+          </div>
+          <div class="border-box">
+            <i class="fas fa-user-graduate"></i>
+            <h3>Scholarship</h3>
+            <p>
+              Scholarship helps you to achieve your goals and many scholarship
+              here you can choose from them.
+            </p>
+          </div>
+        </div>
+        <div class="learn-more">
+          <a class="learn btn btn-primary" href="#" rel="noreferrer noopener"
+            >More Information</a
+          >
+        </div>
+        <div class="see-det">
+          <h3 class="see-de h3">See All Courses</h3>
+        </div>
+      </div>
+    </section>
+    <section class="feature-speak col-12">
+      <h3 class="feature h3">
+        Featured Speakers
+        <span class="under-ll"></span>
+      </h3>
+      <div class="speaker-info col"></div>
+      <button
+        class="butt"
+        type="submit"
+        id="showHideButton"
+        onclick="showImage();"
+      >
+        MORE <i class="fas fa-angle-down"></i>
+      </button>
+      <div id="more-detail">
+        <div class="detail showHide">
+          <img
+            class="speaker-img img-sp"
+            src="./src-image/salma.jpg"
+            alt="salima"
+          />
+          <h3 class="speaker-name spe-na">Salima Ikram</h3>
+          <p class="speaker-desc spea-de">
+            Distinguished University Professor of Egyptology at The American
+            University in Cairo
+          </p>
+          <p class="speaker-mor spea-more">
+            worked as an archaeologist in Turkey, Sudan, Greece and the United
+            States
+          </p>
+        </div>
+        <div class="detail showHide">
+          <img
+            class="speaker-img img-sp"
+            src="./src-image/nagw.jpg"
+            alt="nagw"
+          />
+          <h3 class="speaker-name spe-na">Nagwa Sherif</h3>
+          <p class="speaker-desc spea-de">
+            Architectural Engineering at The American University in Cairo
+          </p>
+          <p class="speaker-mor spea-more">
+            She holds a PhD degree from University of Montreal, Canada in 1984
+          </p>
+        </div>
+      </div>
+    </section>
```

Comment:
**Suggested Changes**
- [ ] This is not currently aligning well.

Your Design
![speackers](https://user-images.githubusercontent.com/7220167/143529416-f85652f1-3165-43f3-b222-4381614be607.png)

Provided Design
![real-speakers](https://user-images.githubusercontent.com/7220167/143529497-152889c7-a539-4e28-81b2-22bfddb7f890.png)